source/app/blueprints/case/templates/case.html

```diff
@@ -176,14 +176,21 @@ <h5>Select activity report template</h5>
                             aria-hidden="true">&times;</span></button>
                       </div>
                       <div class="modal-body">
-                          <div class="col">
-                            <p>Since IRIS v2.0.0, the report generation supports images. Integration of images might fail depending on the situation.<br/><code>Safe Mode</code> can be used to generate the report without them.</p>
-                          </div>
-                          <select class="selectpicker form-control bg-outline-success dropdown-submenu mb-2" data-show-subtext="true" data-live-search="true" id="select_report_act">
-                                {% for report in reports_act %}
-                                      <option data-toggle="tooltip"  value="{{ report[0] }}"  data-subtext="{{ report[3] }}">{{ report[1] }} ({{ report[2].capitalize() }})</option>
-                                {% endfor %}
-                          </select>
+                          {% if reports| length == 0 %}
+                              <div class="alert alert-warning" role="alert">
+                                  <h4 class="alert-heading">No report template found</h4>
+                                  <p>Report templates are configured in <a href="/manage/templates?cid={{case.case_id}}">the management section</a>.</p>
+                              </div>
+                          {% else %}
+                              <div class="col">
+                                <p>Since IRIS v2.0.0, the report generation supports images. Integration of images might fail depending on the situation.<br/><code>Safe Mode</code> can be used to generate the report without them.</p>
+                              </div>
+                              <select class="selectpicker form-control bg-outline-success dropdown-submenu mb-2" data-show-subtext="true" data-live-search="true" id="select_report_act">
+                                    {% for report in reports_act %}
+                                          <option data-toggle="tooltip"  value="{{ report[0] }}"  data-subtext="{{ report[3] }}">{{ report[1] }} ({{ report[2].capitalize() }})</option>
+                                    {% endfor %}
+                              </select>
+                          {% endif %}
```

Comment:
Here {% endif %} is misplaced, the Generate buttons below are shown when there are no template available. It should be at line 209/210 after the generate modal.

Reply:
Moreover, gen_act_report(false); is printed two times, is it correct? Shouldn't it be gen_act_report(true); for the safe generation button?